hack/scripts/releaser/releaser.go

```diff
@@ -114,21 +124,6 @@ func (r *Release) Run() error {
 		return errors.Wrap(err, "Failed to populate current and target versions")
```

Comment:
`populateCurrentAndTargetVersions` can be renamed to `populateVerstionAndSaveReleaseInfo`

Reply:
@TomerShor we don't save info if path isn't passed